docs/REST-API-Reference.md

```diff
@@ -885,4 +885,4 @@ Follows the bulk-update convention in the CodaLab API, but currently only
 allows one update at a time.
 
 
-&uarr; [Back to Top](#table-of-contents)
+&uarr; [Back to Top](#table-of-contents)
```

Comment:
This file is auto-generated by a script...check this still works and is pointed at the right filename.

Reply:
Fixed that in a765472f1c552aed163b6f28f18cce27f48dfe06. How often do we run this script?